gradle.properties

```diff
@@ -6,7 +6,7 @@ pluginRepositoryUrl=https://github.com/SchwarzIT/spectral-intellij-plugin
 pluginVersion=2.0.0
```

Comment:
Happy to do that. Before I do increment to 2.1.0, how do you feel about making failing to run the spectral sub-process an error rather than a warning. Currently if it fails due to invalid ruleset URL or any other issue with executing spectral there is no UI feedback.

Reply:
Version and CHANGELOG updated. Made failed execution an error so user gets some feedback. It's not ideal, but I'm happy to look at adding some cleaner checks for misconfiguration like making sure ruleset file/URL is loadable in context of the configuration panel.